scapy/contrib/gtp.py

```diff
@@ -193,13 +193,13 @@ def answers(self, other):
                 self.version == other.version and
                 self.payload.answers(other.payload))
 
-    @classmethod
-    def dispatch_hook(cls, _pkt=None, *args, **kargs):
-        if _pkt and len(_pkt) >= 1:
-            if (struct.unpack("B", _pkt[0])[0] >> 5) & 0x7 == 2:
-                import gtp_v2
-                return gtp_v2.GTPHeader
-        return GTPHeader
+    @classmethod

```

Comment:
Well, I don't know why ^M keeps showing up on the file, when I go editing it, it doesn't show up, any suggestions?